### PR TITLE
Changes the happy-cluwne mask code to pre-determine cluwne logic

### PR DESCRIPTION
**Pull request by AsV9**
Closes #2514 because this does the same thing but better in my opinion because the cluwne mask retains it's risk factor.

#### Changelog

:cl:   
bugfix: Fixes cluwne mask logic so that you fuckers no longer can take the mask on and off to become a cluwne. Good day.
/:cl:

### CODE REVIEW

File: yogstation/code/modules/clothing/mask.dm [hunk 1]
```diff
@@ -61,8 +59,19 @@
 	desc = "The mask of a poor cluwne that has been scrubbed of its curse by the Nanotrasen supernatural machinations division. Guaranteed to be %99 curse free and %99.9 not haunted. "
 	flags_1 = MASKINTERNALS
 	alternate_screams = list('yogstation/sound/voice/cluwnelaugh1.ogg','yogstation/sound/voice/cluwnelaugh2.ogg','yogstation/sound/voice/cluwnelaugh3.ogg')
-	var/can_cluwne = TRUE
 	item_flags = ABSTRACT
+	var/can_cluwne = TRUE
+	var/is_cursed = FALSE //i don't care that this is *slightly* memory wasteful, it's just one more byte and it's not like some madman is going to spawn thousands of these
+	var/is_very_cursed = FALSE
+
+/obj/item/clothing/mask/yogs/cluwne/happy_cluwne/New()
```
Comment (Amelia0010) [inline yogstation/code/modules/clothing/mask.dm]: /obj/item/clothing/mask/yogs/cluwne/happy_cluwne/Initialize()

File: yogstation/code/modules/clothing/mask.dm [hunk 2]
```diff
@@ -61,8 +59,19 @@
...
+	..()
```
Comment (Amelia0010) [inline yogstation/code/modules/clothing/mask.dm]: .=..()

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

**Comment by AsV9** [comment]
It doesn't functionally change anything though @nichlas0010

**Comment by Amelia0010** [comment]
nope

**Pull request merged**